packet/packet.go

```diff
@@ -135,17 +135,15 @@ func SetCC(packet *Packet, newCC uint8) *Packet {
 	return &newPacket
 }
 
-// Returns a byte slice containing the PES header if the Packet contains one,
+// PESHeader returns a byte slice containing the PES header if the Packet contains one,
 // otherwise returns an error
 func PESHeader(packet *Packet) ([]byte, error) {
 	if ContainsPayload(packet) && PayloadUnitStartIndicator(packet) {
```

Comment:
`Payload()` runs `ContainsPayload()` for us, so this check is no longer necessary.

Reply:
Good catch. I've removed it.